flake.nix

```diff
@@ -131,6 +131,7 @@
             pkgs = nixpkgs.legacyPackages.${system}.extend self.overlays.default;
             lib = nixpkgs.lib;
             # Keep debug symbols disabled for very large packages to avoid long compilation times.
+            # Also, keep it disabled for libraries that do not support overriding (like libgcc).
```

Comment:
Hmm, I guess this comment is outdated now, right?

Reply:
I mean, I guess it can stay there for future reference in case we ever have to add something that doesn't support overriding again.